ReleaseTooling/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -482,16 +482,11 @@ enum CocoaPodUtils {
     for pod in pods {
       let rootPod = String(pod.name.split(separator: "/").first!)
 
-      let podIsClosedSource: Bool = FirebaseManifest.shared.pods
-        .filter(\.isClosedSource)
-        .map(\.name)
-        .contains(rootPod)
-
       // Check if we want to use a local version of the podspec.
-      if !podIsClosedSource,
-         let localURL = localPodspecPath,
-         FileManager.default
-         .fileExists(atPath: localURL.appendingPathComponent("\(rootPod).podspec").path) {
+      if let localURL = localPodspecPath,
+         let pathURL = localPodspecPath?.appendingPathComponent("\(rootPod).podspec").path,
```

Comment:
optional nit: the `localURL` could just be reused since you know it's there by the time `pathURL` is assigned. not a big deal though.

```suggestion
         let pathURL = localURL.appendingPathComponent("\(rootPod).podspec").path,
```

Reply:
Would be nice, but

![Screen Shot 2022-08-24 at 9 24 55 PM](https://user-images.githubusercontent.com/73870/186574756-e57a8daa-cf3e-43d3-8642-b6fdc3c37ce6.png)

It feels like there should be a way to do this, but I'm not sure what it would be ....